src/browser/modules/Sidebar/ProjectsFilesScripts.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2002-2020 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ * This file is part of Neo4j.
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+import React from 'react'
+import { useEffect, useState } from 'react'
+import { withBus } from 'react-suber'
+import { connect } from 'react-redux'
+import MyScripts from '@relate-by-ui/saved-scripts'
+import { useQuery, useMutation, ApolloError } from '@apollo/client'
+import path from 'path'
+import { filter, size, omit } from 'lodash-es'
+
+import * as editor from 'shared/modules/editor/editorDuck'
+import { executeCommand } from 'shared/modules/commands/commandsDuck'
+import { SLASH, CYPHER_FILE_EXTENSION } from 'shared/services/export-favorites'
+import {
+  ProjectFilesQueryVars,
+  ProjectFileMutationVars,
+  mapProjectFileToFavorites,
+  updateCacheRemoveProjectFile
+} from './project-files.utils'
+import {
+  IFavorite,
+  IProjectFilesResult,
+  IProjectFilesVariables,
+  SELECT_PROJECT_FILE,
+  GET_PROJECT_FILES,
+  DELETE_PROJECT_FILE,
+  REMOVE_PROJECT_FILE
+} from './project-files.constants'
+import Render from 'browser-components/Render'
+import { Bus } from 'suber'
+
+interface IProjectFilesError {
+  apolloErrors: (ApolloError | undefined)[]
+  errors?: string[]
+}
+
+export const ProjectFilesError = ({
+  apolloErrors,
+  errors
+}: IProjectFilesError): JSX.Element => {
+  const definedApolloErrors = filter(
+    apolloErrors,
+    apolloError => apolloError !== undefined
+  )
+  const definedErrors = filter(errors, error => error !== '')
+  const ErrorsList = () => {
+    if (size(definedErrors)) {
+      return definedErrors.map((definedError, i) => (
+        <span key={`${definedError}-${i}`}>{definedError}</span>
```

Comment:
Do we show only "normal" errors when there are both apollo and normal errors because they're more important?

Reply:
let me check this

Reply:
Im gonna adjust this slightly so will show both